src/main/resources/application.properties

```diff
@@ -5,6 +5,7 @@ spring.jpa.database=POSTGRESQL
 spring.jpa.hibernate.ddl-auto=validate
 spring.jpa.hibernate.use-new-id-generator-mappings=true
 spring.datasource.url=jdbc:postgresql://localhost:5432/local_parrit
+spring.datasource.username=postgres
```

Comment:
I don't think this is necessary.  By default it should use your computer's name as the username, which shouldn't be an issue.  Or did you have other problems?

Reply:
Accidentally committed this. But I need it for it to work on my machine. Standard postgres installation on Arch Linux. If this doesn't make it fail on yours it'd be cool to have since I have to have this local change all the time.

Reply:
It would make it fail on everyone's if they've already created a db.  One thing we had before was forcing everyone to create a "parrit" user in postgres, and create the "local_parrit" db using that user and setting `spring.datasource.username` to "parrit".  But was a lot of work for what didn't needed to be done for a lot of people.